contrib/gitlab.tpl

```diff
@@ -1,10 +1,11 @@
 {{- /* Template based on https://docs.gitlab.com/ee/user/application_security/container_scanning/#reports-json-format */ -}}
 {
-  "version": "2.3",
+  "version": "14.0.6",
   "vulnerabilities": [
   {{- $t_first := true }}
   {{- range . }}
   {{- $target := .Target }}
+    {{ $image := $target | regexFind "[^\\s]+" }}
```

Comment:
Not exactly a valid docker image string either, but `14.0.6` doesn't validate the pattern, so we can be less strict here.